### DISCUSSION TITLE
[求助]有关首页标语动画自定义字符分割问题

### DISCUSSION THREAD

**Discussion by leitingsd** [YunYouJun/valaxy#319] [Q&A] [answered]
RT，首页的垂直交错排列标语动画现在如何实现自定义字符分割，而不是一个字符为一行的原版分割呢？
现版本的文档好像没有提到有关这个标语动画组件设置相关的内容，Hexo-Theme-Yun主题文档有关该组件的部分提到的数组形式来自定义字符分割的方法在现版本不可用

**Comment by leitingsd** [comment, marked as answer]
已解决，帖子关闭
实现自定义分割的方式为
`title: ['第一行', '第二行', '第三行', '第四行'],`
以此类推。

建议将有关这个组件的新使用方式和说明更新进文档，YunBanner.vue指向的是hexo-theme-yun，而其[文档](https://yun.yunyoujun.cn/guide/config.html#%E6%A0%87%E8%AF%AD%E5%8A%A8%E7%94%BB)上有关该组件的自定义分割方式已不适用于现版本。

**Reply by YunYouJun** [reply]
Yaml 语法 `-` 对应的即数组形式。

已在 valaxy-theme-yun 文档中添加注释。
